tests/datasets/test_datasets.py

```diff
@@ -109,6 +110,30 @@ def test_blogcatalog3_deprecated_load() -> None:
         load_dataset_BlogCatalog3(dataset.data_directory)
 
 
+def test_mutag_load() -> None:
+    graphs, labels = MUTAG().load()
+
+    n_graphs = 188
+
+    assert len(graphs) == n_graphs
+    assert len(labels) == n_graphs  # one label per graph
+
+    # get a list with the number of nodes in each graph
```

Comment:
FWIW, I'm unsure of the value of comments like this, because they just rephrase what the code is saying. It's definitely valuable if the code is subtle, but most of the code here is straightforward. Explaining what the (exact) average is definitely useful, but (effectively) explaining that `np.mean` calculates an average doesn't seem quite as useful.